ui/app/components/secure-variable-form.js

```diff
@@ -87,9 +88,27 @@ export default class SecureVariableFormComponent extends Component {
   @action
   async save(e) {
     e.preventDefault();
-    this.args.model.set('keyValues', this.keyValues);
-    this.args.model.setAndTrimPath();
-    await this.args.model.save();
+    try {
+      this.args.model.set('keyValues', this.keyValues);
+      this.args.model.setAndTrimPath();
+      await this.args.model.save();
+      this.flashMessages.add({
+        title: 'Secure Variable saved',
+        message: `${this.args.model.path} successfully saved`,
+        type: 'success',
+        destroyOnClick: false,
+        timeout: 5000,
```

Comment:
`question` `performance (testing)`: Will this block the thread of execution in our test suites? I could imagine our test suite being slowed down waiting for this flash message to disappear. Should we consider `timeout` to read from our `config` file that can determine if we're in `dev`, `prod` or `testing` and appropriately set the timer?

Reply:
No, these are considered non-blocking; they happen asynchronously and allow transitions and anything else to happen aside from them.